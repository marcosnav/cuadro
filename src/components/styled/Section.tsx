import styled from 'styled-components';

const Section = styled.section`
  margin-top: 42px;
  text-align: center;
`;

Section.displayName = 'Section';

export default Section;
