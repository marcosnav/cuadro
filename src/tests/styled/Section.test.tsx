import { mount } from 'enzyme';
import 'jest-styled-components';
import React from 'react';
import Section from './../../components/styled/Section';

describe('<Section/>', () => {
  test('renders properly', () => {
    const component = mount(<Section>section text</Section>);
    expect(component.text()).toBe('section text');
    expect(component).toHaveStyleRule('margin-top', '42px');
    expect(component).toHaveStyleRule('text-align', 'center');
    expect(component).toMatchSnapshot();
  });
});
