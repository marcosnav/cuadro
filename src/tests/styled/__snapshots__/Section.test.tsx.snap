// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section/> renders properly 1`] = `
.c0 {
  margin-top: 42px;
  text-align: center;
}

<Section>
  <section
    className="c0"
  >
    section text
  </section>
</Section>
`;
